src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -672,15 +680,56 @@ def image_mask_iterator():
                     description="1 if ROI was rejected or 0 if accepted as a cell during segmentation operation",
                 ),
             ],
-            id=roi_ids,
         )
+        if mask_type != "pixel" and mask_type != "voxel":  # roi_ids will not be written row-wise
+            plane_segmentation_kwargs.update(id=roi_ids)
```

Comment:
This is only necessary due to the mixed column-wise/row-wise writing below

Reply:
You can add this together in the other if in line 675 I think.